From ec97e5c43e869898f1a1f0d00ac5715c1edd06f1 Mon Sep 17 00:00:00 2001
From: Marc Hanheide <marc@hanheide.net>
Date: Wed, 25 Feb 2015 13:37:57 +0000
Subject: [PATCH] re-added strip exclusion

---
 debian/rules.em |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules.em b/debian/rules.em
index 991c991..5c0bdd6 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -60,3 +60,6 @@ override_dh_auto_install:
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_install
+
+override_dh_strip:
+	dh_strip -Xlibav
-- 
1.7.9.5

